templates/firstboot-nvme.yaml.j2

```diff
@@ -29,7 +29,7 @@ resources:
           template: |
             #!/bin/bash
             set -x
-            FORMAT='compute'
+            FORMAT='pci'
```

Comment:
This will break mount_nvme now. You are checking for pci in hostname, which will fail for compute devices which wants to mount nvme instead of passthrough.